ingress/helm-agic-config-template.txt

```diff
@@ -51,5 +51,5 @@ rbac:
     enabled: true # true/false
 
 # Specify aks cluster related information. THIS IS BEING DEPRECATED.
-aksClusterConfiguration:
-    apiServerAddress: ${apiAddress}
+#aksClusterConfiguration:
+#    apiServerAddress: ${apiAddress}
```

Comment:
Is the plan to reinstate these commented-out lines at some point?

Reply:
Stripped out of template - old config